src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -484,6 +484,46 @@ def stop_leftover_processes(self, processes, hosts):
                 hosts, ",".join(processes))
             stop_processes(hosts, "'({})'".format("|".join(processes)))
 
+    def check_pool_free_space(self, pool, expected_scm=None, expected_nvme=None,
+                              timeout=30):
+        """Check pool free space with expected value.
+        Args:
+            pool (TestPool): The pool for which to check free space.
+            expected_scm (int, optional): pool expected SCM free space.
+            expected_nvme (int, optional): pool expected NVME free space.
+            timeout(int, optional): time to fail test if it could not match
+                expected values.
+        Note:
+            Arguments may also be provided as a string with a number preceded
+            by '<', '<=', '>', or '>=' for other comparisons besides the
+            default '=='.
+        """
+        if not expected_scm and not expected_nvme:
+            self.fail("at least one space parameter must be specified")
+        done = False
+        scm_fs = 0
+        nvme_fs = 0
+        start = time()
+        scm_index, nvme_index = 0, 1
+        while time() - start < timeout:
+            sleep(1)
+            checks = []
+            pool.get_info()
+            scm_fs = pool.info.pi_space.ps_space.s_free[scm_index]
+            nvme_fs = pool.info.pi_space.ps_space.s_free[nvme_index]
+            if expected_scm is not None:
+                checks.append(("scm", scm_fs, expected_scm))
+            if expected_nvme is not None:
+                checks.append(("nvme", nvme_fs, expected_nvme))
+            done = pool._check_info(checks)
```

Comment:
(pylint-protected-access) Access to a protected member _check_info of a client class